python_modules/libraries/dagster-datahub/setup.py

```diff
@@ -40,6 +40,7 @@ def get_version() -> str:
         f"dagster{pin}",
         "packaging",
         "requests",
+        "confluent-kafka<2.8.1",
```

Comment:
maybe != unless you trust yourself to undo this once wheels are up

or maybe stack a PR no so you dont forget?